api/src/main/java/org/opencord/aaa/AuthenticationStatisticsService.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018-present Open Networking Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.opencord.aaa;
+
+import org.onosproject.event.ListenerService;
+
+/**
+ * Service for interacting with accounting module.
+ */
+
+public interface AuthenticationStatisticsService extends
+        ListenerService<AuthenticationStatisticsEvent, AuthenticationStatisticsEventListener> {
+    public AaaStatistics getAaaStats();
+    public AuthenticationStatisticsDelegate getStatsDelegate();
```

Comment:
Only expose relevant methods from this Service. I guess only getAaaStats should be defined in this interface as that will be consumed in KafkaOnos app. All other 3 methods are used with this AAA application and hence should be private methods in AaaStatisticsManager class